tycho-indexer/src/main.rs

```diff
@@ -302,6 +302,15 @@ async fn run_rpc(global_args: GlobalArgs) -> Result<(), ExtractionError> {
         .build_gw()
         .await?;
 
+    // TODO without this line I get Unexpected storage error: Usage error: No transaction started%
+    //  Calling this line solves the issue.
+    //  I can't put this inside the RPCHandler since it takes a Gateway - not a CachedGateway
+    //  generic - is it fine I change that?
+    //  Note to self: I guess I also have to commit after?
+    // cached_gw
+    //     .start_transaction(&Block::default(), None)
+    //     .await;
+
```

Comment:
Need an opinion here - is it fine if I change the generic trait G in `rpc.rs` to `CachedGateway` instead of `Gateway`? `start_transaction` doesn't exist for just `Gateway`. Or should I move the `start_transaction` to here?